docs/source/tutorials/token_classification/rank.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd 
+import numpy as np 
+from cleanlab.rank import get_label_quality_scores as main_get_label_quality_scores 
+
+def get_label_quality_scores(labels: list, 
+                             pred_probs: list, 
+                             method: str = "min", 
+                             cleanlab_method: str = "self_confidence", 
```

Comment:
Let's change the order of these two args and rename:
`method` -> `pool_token_scores_method`
`cleanlab_method` -> `method` (to match the existing `rank.get_label_quality_scores()` API).